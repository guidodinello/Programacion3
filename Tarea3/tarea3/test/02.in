leer_dias 6
leer_precios 2 7 6 1 8 10
leer_disparos 2
k_disparos
Fin