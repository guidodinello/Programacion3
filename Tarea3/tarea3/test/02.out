1>2>3>4>Con precios: 2 7 6 1 8 10 
La ganancia maxima con 2 disparo(s) es: 14000 
5>Fin.
